README.md

```diff
@@ -50,4 +50,21 @@ Filter type can be rectangular or round(in future maybe shaped)
     RectangularFilter {
         point top left, point bottom right {latitude, longitude}   // A couple of points on a map that describes margin of rectangle area.
     }
-```
+```  
+
+## MIGRATIONS
+
+Migrate with using golang/migrate default table/rules for user
+In table must be created default user with name 'fuser' without any rules with own password
```

Comment:
WDYT about renaming 'fuser' to 'pf_user'?
Pattern: [app abbreviation] + '_user'

Reply:
renamed